### PR TITLE
Remove vaultPath from motion apis as the secret provider has been rem…

### PR DESCRIPTION
**Pull request by froesef**
…oved

This is a follow-up on #432 for the motion APIs

### CODE REVIEW

File: manager/testdata/blueprint.yaml [hunk 1]
```diff
@@ -27,7 +27,8 @@ spec:
                 endpoint: s3.eu-gb.cloud-object-storage.appdomain.cloud
                 object_key: small.kp.encrypted.parq
             type: 2
-            credentialLocation: /v1/m4d/dataset-creds/m4d-test-bucket
+            vault:
+              secretPath: /v1/m4d/dataset-creds/m4d-test-bucket
```
Comment (revit13) [inline manager/testdata/blueprint.yaml:31]: vault fields also includes the following sub-feilds:
   vault:
      address: http://vault.m4d-system:8200
      authPath: /v1/auth/kubernetes/login
      role: module
      secretPath: "/v1/kubernetes-secrets/secret-name?namespace=default"

File: manager/testdata/blueprint.yaml [hunk 2]
```diff
@@ -52,7 +53,8 @@ spec:
                   endpoint: s3.eu-gb.cloud-object-storage.appdomain.cloud
                   object_key: small.kp.encrypted.parq
                 type: 2
-              credentialLocation: /v1/m4d/dataset-creds/m4d-test-bucket
+              vault:
+                secretPath: /v1/m4d/dataset-creds/m4d-test-bucket
```
Comment (revit13) [inline manager/testdata/blueprint.yaml:57]: Same here.

File: manager/testdata/plotter.yaml [hunk 3]
```diff
@@ -36,7 +37,8 @@ spec:
                     database: MYDB
                     url: mydb
                     ssl: "true"
-                credentialLocation: "/v1/secret/data/db2/m4d"
+                vault:
+                  secretPath: "/v1/secret/data/db2/m4d"
```
Comment (revit13) [inline manager/testdata/plotter.yaml:41]: Same here.

File: manager/testdata/plotter.yaml [hunk 4]
```diff
@@ -26,7 +26,8 @@ spec:
                     endpoint: s3.eu-gb.cloud-object-storage.appdomain.cloud
                     object_key: my-object
                   type: 2
-                credentialLocation: /v1/m4d/dataset-creds/m4d-test-bucket
+                vault:
+                  secretPath: /v1/m4d/dataset-creds/m4d-test-bucket
```
Comment (revit13) [inline manager/testdata/plotter.yaml:30]: Same here

File: modules/m4d-implicit-copy-batch/values.yaml.sample [hunk 5]
```diff
@@ -12,7 +12,6 @@ copy:
         endpoint: s3.eu-gb.cloud-object-storage.appdomain.cloud
         object_key: car.data.copy.parq
       type: 2
-    credentialLocation: /v1/m4d/dataset-creds/m4d-test-bucket
```
Comment (revit13) [inline modules/m4d-implicit-copy-batch/values.yaml.sample:15]: The vault field can be added here.

File: modules/m4d-implicit-copy-batch/values.yaml.sample [hunk 6]
```diff
@@ -23,7 +22,6 @@ copy:
         endpoint: s3.eu-gb.cloud-object-storage.appdomain.cloud
         object_key: car.data.parq
       type: 2
-    credentialLocation: /v1/m4d/dataset-creds/m4d-test-bucket
```
Comment (revit13) [inline modules/m4d-implicit-copy-batch/values.yaml.sample:26]: Same here.

File: modules/m4d-implicit-copy-stream/values.yaml.sample [hunk 7]
```diff
@@ -18,7 +18,6 @@ copy:
         url: dashdb-txn-sbox-yp-lon02-02.services.eu-gb.bluemix.net
       name: db2
       type: 3
-    credentialLocation: /v1/m4d/dataset-creds/87ffdca3-8b5d-4f77-99f9-0cb1fba1f73f/01c6f0f0-9ffe-4ccc-ac07-409523755e72
```
Comment (revit13) [inline modules/m4d-implicit-copy-stream/values.yaml.sample:21]: Same here.

File: modules/m4d-implicit-copy-stream/values.yaml.sample [hunk 8]
```diff
@@ -29,7 +28,6 @@ copy:
         endpoint: s3.eu-gb.cloud-object-storage.appdomain.cloud
         object_key: car.data.parq
       type: 2
-    credentialLocation: /v1/m4d/dataset-creds/m4d-test-bucket
```
Comment (revit13) [inline modules/m4d-implicit-copy-stream/values.yaml.sample:32]: Same here.

**Review by roee88 — APPROVED** [review]

**Pull request merged**